ci/test/06_script_a.sh

```diff
@@ -6,7 +6,10 @@
 
 export LC_ALL=C.UTF-8
 
-BITCOIN_CONFIG_ALL="--disable-dependency-tracking --prefix=$DEPENDS_DIR/$HOST --bindir=$BASE_OUTDIR/bin --libdir=$BASE_OUTDIR/lib"
+BITCOIN_CONFIG_ALL="--enable-suppress-external-warnings --disable-dependency-tracking --prefix=$DEPENDS_DIR/$HOST --bindir=$BASE_OUTDIR/bin --libdir=$BASE_OUTDIR/lib"
```

Comment:
should the diff in this line (9) be part of the first commit?

Reply:
Why do you think so? The first commit seems complete to me, and it shouldn't fire any error on CI, no?

Reply:
Ok, then I think they should be a separate pull request: https://github.com/bitcoin/bitcoin/pull/20182#issuecomment-713425454

Doing build system changes under the `ci: ` prefix seems confusing.

Reply:
Done in #20544.

Reply:
20544 fails without `--enable-suppress-external-warnings`:

```
/tmp/cirrus-ci-build/depends/arm-linux-gnueabihf/include/boost/asio/impl/error.ipp:102:15: error:   by ‘virtual std::__cxx11::string boost::asio::error::detail::misc_category::message(int) const’ [-Werror=overloaded-virtual]
   std::string message(int value) const
               ^~~~~~~

Reply:
> should the diff in this line (9) be part of the first commit?

You're right. Some jobs already have `--enable-werror` :man_facepalming: